appnav/src/main/kotlin/io/element/android/appnav/loggedin/SendQueues.kt

```diff
@@ -29,18 +29,19 @@ class SendQueues @Inject constructor(
     private val matrixClient: MatrixClient,
     private val networkMonitor: NetworkMonitor,
 ) {
+    /**
+     * Launches the send queues retry mechanism in the given [coroutineScope].
+     * Makes sure to re-enable all send queues when the network status is [NetworkStatus.Online].
+     */
+    @OptIn(FlowPreview::class)
     fun launchIn(coroutineScope: CoroutineScope) {
-        networkMonitor.connectivity
-            .onEach { networkStatus ->
-                matrixClient.setAllSendQueuesEnabled(enabled = networkStatus == NetworkStatus.Online)
-            }
-            .launchIn(coroutineScope)
-
-        @OptIn(FlowPreview::class)
-        matrixClient.sendQueueDisabledFlow()
+        combine(
+            networkMonitor.connectivity,
+            matrixClient.sendQueueDisabledFlow(),
```

Comment:
Why do we need to listen to `matrixClient.sendQueueDisabledFlow()` here?

Reply:
If the send queue get disabled for some reason, we want to be aware so we can enable it back (could be the case for exemple when restarting the app)

Reply:
OK, thanks, you can merge!